servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -173,8 +173,10 @@ nessie.server.authentication.anonymous-paths=/q/health/live,/q/health/live/,/q/h
 # to be overwritten by end user when enabling OpenID validation
 quarkus.oidc.auth-server-url=http://127.255.0.0:0/auth/realms/unset/
 quarkus.http.auth.basic=false
-# OIDC-enabled is a build-time property (cannot be overwritten at run-time), MUST be true
+# OIDC-enabled is a build-time property (cannot be overwritten at run-time), MUST be true.
+# However, we can overwrite the tenant-enabled property at run-time.
 quarkus.oidc.enabled=true
+quarkus.oidc.tenant-enabled=${nessie.server.authentication.enabled}
```

Comment:
Do we need the default value for `quarkus.oidc.auth-server-url` not?

Reply:
Good point, I think we can (should) get rid of it.

Reply:
+1 from me on both points